google/auth/transport/requests.py

```diff
@@ -332,6 +346,7 @@ def request(self, method, url, data=None, headers=None, timeout=None, **kwargs):
                 url,
```

Comment:
Could you modify the comment on Line 343 to say 'adjusted max_allowed_time' (or something along those lines) rather than 'adjusted timeout'?

GitHub won't let me comment on the line itself :sweat_smile:

Reply:
Done, thanks.